app/controllers/api/v0/analytics_controller.rb

```diff
@@ -0,0 +1,66 @@
+module Api
+  module V0
+    include Pundit
```

Comment:
this includes Pundit in the whole V0 module, so the other controllers as well, dunno if this is intended behavior or not

Reply:
Oh great point, meant to remove that.